<?xml version="1.0" encoding="UTF-8"?>
<launch>
      
  <!-- Load joint controllers -->
  <rosparam file="$(find robot1)/config/trajectory_control.yaml" command="load" />
  <!-- Joint Controllers -->
  <node name="robot_er_controller_spawner" pkg="controller_manager" type="spawner"
        respawn="false" output="screen" ns="/robot_er"
        args="robot_er_joint_controller"/>
        
  <!--node name="rqt_publisher" pkg="rqt_publisher" type="rqt_publisher" /-->

  <!-- Load controllers -->
  <rosparam file="$(find robot1)/config/joints_states.yaml" command="load" />
  <!-- Controllers -->
  <node name="joint_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" 
        output="screen" ns="/robot_er" args="joint_state_controller"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
        respawn ="false" output="screen" >
    <remap from="/joint_states" to="/robot_er/joint_states" />
  </node>

  <rosparam file="$(find robot1)/config/gazebo_ros_control_params.yaml" command="load"/>

  <!--node pkg="rviz" type="rviz" name="rviz" required="true"
        args="-d $(find robot1)/rviz/robot_er.rviz"/-->
          
</launch>
